plugins/pipelines/isAdmin.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const idSchema = joi.reach(schema.models.pipeline.base, 'id');
+
+module.exports = () => ({
+    method: 'GET',
+    path: '/pipelines/{id}/isAdmin',
```

Comment:
wonder if `isAdmin` is confusing, since you can be a Screwdriver admin and/or a pipeline admin
should be okay since the route is prefaced by `/pipeline`

Reply:
Changing the endpoint since we need it for jobs and events as well